### PR TITLE
BF: Cast integer session identifiers to strings.

### PR DESCRIPTION
**Pull request by arokem**
Before concatenating with subject identifiers.

### CODE REVIEW

File: afqinsight/tests/test_transform.py [hunk 1]
```diff
@@ -41,7 +44,7 @@ def test_AFQDataFrameMapper(concat_subject_session):
     assert cols == cols_ref  # nosec
     if concat_subject_session:
         assert set(subjects) == set(
-            (nodes.subjectID + nodes.sessionID).unique()
+            (nodes.subjectID + nodes.sessionID.astype(str)).unique()
```
Comment (arokem) [inline afqinsight/tests/test_transform.py:47]: Now when I think about it, should we also cast `nodes.subjectID` as string here? I guess subject identifiers can also be integers, which would raise the same error here.

**Comment by arokem** [comment]
[This CI error](https://github.com/richford/AFQ-Insight/runs/4202471654?check_suite_focus=true#step:5:8) is weird. The Black documentation suggests that it should be ignoring files that are in the .gitignore.

**Comment by arokem** [comment]
Also, [this](https://github.com/richford/AFQ-Insight/runs/4205350298?check_suite_focus=true#step:5:98) doc build error is a bit mysterious, but maybe we should just drop 3.6 support (and add 3.9?)

File: afqinsight/tests/test_transform.py (hunk 1 above)
Comment (richford) [reply afqinsight/tests/test_transform.py:47]: Yeah, good catch. Thanks!

**Comment by richford** [comment]
Yes, I'm +1 for dropping 3.6 support and adding 3.9.

**Comment by richford** [comment]
I too am perplexed by the linting issue. I don't understand why it's not ignoring that file. And why it did ignore that file in CI tests for #90.

**Pull request merged**